src/main/scala/com/linagora/openpaas/gatling/calendarAndContact/scenari/CalendarAndContactsScenari.scala

```diff
@@ -37,8 +35,7 @@ object CalendarAndContactsScenari {
         20.0 -> {
           exec(AddressBookSteps.openContactsSpa())
             .during(ScenarioDuration) {
-              exec(PKCESteps.renewAccessToken)
-                .randomSwitch(
+                randomSwitch(
```

Comment:
Thank you for the catch, it is the other way around, i removed the explicit call to renewToken in the scenario because it is now part of Authentication.withAuth so it is called before every authenticated requests